solc_json_parser/parser.py

```diff
@@ -481,6 +496,12 @@ def function_by_name(self, contract_name: str, function_name: str) -> Function:
         contract = self.contract_by_name(contract_name)
         funcs    = self.functions_in_contract(contract)
         return next(fn for fn in funcs if fn.name == function_name)
+    
+    def all_libraries(self) -> List[ContractData]:
+        return [contract for contract in self.all_contracts() if contract.kind == "library"]
+    
+    def all_libraries_names(self) -> List[str]:
```

Comment:
Please use `@cached_property`, so we don't need to create a list everytime calling this method